ruma-serde/src/canonical_json.rs

```diff
@@ -1,10 +1,12 @@
-use std::fmt;
+use std::{collections::BTreeMap, convert::TryInto, fmt};
 
 use serde::Serialize;
-use serde_json::Error as JsonError;
+use serde_json::{Error as JsonError, Map as JsonObject, Value as JsonValue};
 
 pub mod value;
 
+pub type CanonicalJsonObject = BTreeMap<String, value::CanonicalJsonValue>;
```

Comment:
I think we should declare and use this as `value::Object`, re-export it at `ruma_serde::CanonicalJsonObject` and remove the identical definition in `ruma-signatures`.

Reply:
Do we want to still re-export this from `ruma-signatures`?

Reply:
Nope.